### PR TITLE
I AM U

### PR DESCRIPTION
**Pull request by ishan-renjen**
made the IMU repository

### CODE REVIEW

File: include/IMU.hpp [hunk 1]
```diff
@@ -10,4 +10,4 @@ class BOARD_NAME {
 private:
 };
 
-}
+}// namespace BOARD_NAME
```
Comment (AarC10) [inline include/IMU.hpp]: Make sure there is a newline at the end of this file

**Review by AarC10 — CHANGES_REQUESTED** [review]

**Review by Ethangirard62 — APPROVED** [review]

**Review by ActuallyTaylor — APPROVED** [review]

File: include/IMU.hpp (hunk 1 above)
Comment (ActuallyTaylor) [reply include/IMU.hpp]: Should be fixed in the latest commit

**Review by AarC10 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,77 +1,4 @@
...
-confirmed, these files should be deleted.  There are placeholders to demonstrate the board library 
-building functionality.
-8) Everything in this README from this final step up should be deleted, leaving only the content 
-below. When finished, all the changes should be committed and pushed to the setup branch, and a PR
-should be created to merge into main.
-
-# BIKE_NAME-BOARD_NAME
+# DEV1-IMU
 
 ## Introduction
```
Comment (mjmagee991) [inline README.md:4]: Add a short summary of what the IMU does here

File: include/IMU.hpp [hunk 3]
```diff
@@ -10,4 +10,4 @@ class BOARD_NAME {
 private:
```
Comment (mjmagee991) [inline include/IMU.hpp:10]: Everything that says "BOARD_NAME" should be changed to IMU

File: src/IMU.cpp [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#include <IMU.hpp>
+
+namespace BOARD_NAME {}
```
Comment (mjmagee991) [inline src/IMU.cpp]:
```suggestion
namespace IMU {}
```

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Looks good for the most part, but a few things need to be renamed

File: README.md (hunk 2 above)
Comment (mjmagee991) [reply README.md:4]: Talk to me if you're not sure what to put here

**Comment by mjmagee991** [comment]
It looks like the TMU has the same issues. When you get a chance, can you make a branch to fix it there as well?

**Review by mjmagee991 — APPROVED** [review]
Nice work

**Review by ActuallyTaylor — APPROVED** [review]

**Pull request merged**